services/bugzilla/bugzilla.service.js

```diff
@@ -18,8 +18,9 @@ const schema = Joi.object({
     .required(),
 }).required()
 
-const documentation = `
+const description = `
 <p>
+  Use the <code>baseUrl</code> query parameter to target different Bugzilla deployments.
```

Comment:
Unlike most other badges, `baseUrl` is not a required query parameter. However, it's quite easy to miss when it's not marked as required, so decided to add a small bit of text in the example's description.

Alternatively, I could have marked it as required in the example, even though we'd still fall back underneath the hood if not specified.

Let me know what you think.

Reply:
I don't think we should mark optional parameters as required.

What you've done here is totally reasonable. In general, I have not been doing anything extra to draw attention to optional query params. So if you look at https://shields.io/badges/git-lab-release for example, there are some extra notes around `gitlab_url` but there's nothing extra calling out

- `include_prereleases`
- `sort`
- `display_name` or
- `date_order_by`

basides just documenting them. I do think that there is a distinction between

- optional query params that are specific to this badge (e.g: `include_prereleases`) and
- optional query params that can be used with all badges (e.g: `logo`)

and it would be nice to be able to make that clearer. Neither the OpenAPI spec nor the rendering package really gives us a way to make a distinction between them though.

At the moment, I'm just focussing on trying to plough through the data conversion.